content/installation/upgrade.md

```diff
@@ -11,6 +11,13 @@ Custom installations do not support client-side upgrade functionality.
 Use [`kubectl directpv migrate`]({{< relref "/command-line/migrate.md" >}}) to migrate the old resources to a new installation.
 {{< /admonition >}}
 
+{{< admonition title="Pod Security Policies" type="note" >}}
+Kubernetes deprecated the `PodSecurityPolicy` feature in v1.21, then removed it entirely in v1.25.
+The Kubernetes documentation [recommends]](https://kubernetes.io/docs/concepts/security/pod-security-policy/) [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/) as a replacement.
+
+DirectPV continues to support `PodSecurityPolicy` in the 4.0.x lineage.
```

Comment:
lineage -> series ? We might need to decide on terminology

Reply:
Just going with `versions`.